pkg/store/storepb/rpc.proto

```diff
@@ -39,6 +40,21 @@ service Store {
   rpc LabelValues(LabelValuesRequest) returns (LabelValuesResponse);
 }
 
+/// WriteableStore reprents API against instance that stores XOR encoded values with label set metadata (e.g Prometheus metrics).
+service WriteableStore {
+  // WriteRequest allows you to write metrics to this store via remote write
+  rpc RemoteWrite(WriteRequest) returns (WriteResponse) {}
+}
+
+message WriteResponse {
+}
+
+message WriteRequest {
+  string tenant = 1;
+  int64 replica = 2;
+  repeated prometheus.TimeSeries timeseries = 3 [(gogoproto.nullable) = false];
```

Comment:
how about putting this field first so that clients, e.g. a modified prometheus, that only know about the prometheus.TimeSeries field can send to us and we interpret tenant as `''` and replica as `0`?